homeassistant/components/recorder/strings.json

```diff
@@ -12,6 +12,10 @@
     "maria_db_range_index_regression": {
       "title": "Update MariaDB to {min_version} or later resolve a significant performance issue",
       "description": "Older versions of MariaDB suffer from a significant performance regression when retrieving history data or purging the database. Update to MariaDB version {min_version} or later and restart Home Assistant. If you are using the MariaDB core add-on, make sure to update it to the latest version."
+    },
+    "backup_failed_out_of_resources": {
+      "title": "Database backup failed due to lack of resources",
+      "description": "The database backup stated at {start_time} failed due to lack of resources. This can happen if the database is too large or if the system is under heavy load. Consider upgrading the system hardware or reducing the size of the database by decreasing the number of history days to keep or creating a filter."
```

Comment:
Do we need to say database here? There could be a misconfigured, or large, add-on (as an example) that is backed up and consumes all storage resources. All we know is that the backup failed, not that it was caused due to the database. I just feel this could cause some confusion.

Reply:
The rest of the backup is likely fine